source/postprocess/visualization/depth_including_mesh_deformation.cc

```diff
@@ -0,0 +1,82 @@
+/*
+  Copyright (C) 2011 - 2022 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#include <aspect/postprocess/visualization/depth_including_mesh_deformation.h>
+#include <aspect/geometry_model/interface.h>
+
+
+
+namespace aspect
+{
+  namespace Postprocess
+  {
+    namespace VisualizationPostprocessors
+    {
+      template <int dim>
+      DepthIncMeshDef<dim>::
+      DepthIncMeshDef ()
+        :
+        DataPostprocessorScalar<dim> ("depth",
```

Comment:
This is the name of the variable as it will appear in Paraview. Need to make sure it is not duplicated with the other postprocessor.
```suggestion
        DataPostprocessorScalar<dim> ("depth_including_mesh_deformation",
```